examples/guides/mp-restful-webservice/src/main/docs/index.adoc

```diff
@@ -0,0 +1,487 @@
+`getMessage`, and `updateGreeting`. (We annotate `updateGreeting` for simplicity
+and so the metrics
+reported here have the same values as for the Helidon SE RESTful web 
+service example. In a real application we might measure the `update` method separately
+from the `get` methods.)
++
+Your or your IDE will add these imports:
++
+[source,java]
+----
+include::{greet-resource-class}[tag=metricsImports]
+----
++
+[source,java]
+----
+include::{greet-resource-class}[tag=countedAnno]
+----
+<1> Marks this method as measured by a `Counter` metric.
+<2> Declares the unique name for this counter among all metrics.
+<3> Allows the same counter to accumulate uses of multiple methods.
+<4> Indicates that the metrics system should increment the counter on each invocation but
+_not_ decrement it when the method returns. 
+
+=== Rebuild and rerun your application
+Follow the same steps as before, remembering to stop any instance of your application
+that is still running.
+
+include::{se-guide-adoc}[tag=sendRequestsForMetrics]
+
+=== Send some requests
+Use the same `curl` commands from the beginning of the guide to send requests to
+the server:
+
+|====
+|Command 
+a|[source,bash]
+curl -X GET http://localhost:8080/greet
+a|[source,bash]
+curl -X GET http://localhost:8080/greet/Joe
+a|[source,bash]
+curl -X PUT http://localhost:8080/greet/greeting/Hola
+a|[source,bash]
+curl -X GET http://localhost:8080/greet/Jose
+|====
+
+=== Retrieve metrics
+Run this `curl` command to retrieve the collected metrics:
+[source,bash]
+----
+curl -X GET http://localhost:8080/metrics/application <1>
+----
+<1> Requests all application-scoped metrics (we only have one).
+You should see this output (in Prometheus format):
+[listing]
+----
+# TYPE application:io_helidon_guides_mp_restfulwebservice_greet_resource_accessctr counter
+# HELP application:io_helidon_guides_mp_restfulwebservice_greet_resource_accessctr Total greetings accesses
+application:io_helidon_guides_mp_restfulwebservice_greet_resource_accessctr 4
+----
+Note that:
+
+. The name of the counter is automatically qualified with the package and class name of the JAX-RS
+resource that records the metric (`io_helidon_guides_mp_restfulwebservice_greet_resource_accessctr`). If we
+had added `absolute=true` to the `@Counted` annotation attributes then the name would be
+simply `accessctr`.
+
+. The first two lines are gathered from the metadata we included in the `@Counted` 
+annotation.
+
+. As expected, the value for the counter is 4.
+
+A `curl` to `http://localhost:8080/metrics` lists not only our application-scoped 
+metric but all the _base_ and _vendor_ metrics as defined in the MP metrics spec. 
+For example, you will see a `vendor:requests_count` 
+counter. This will be larger than our counter because that counter also tracks 
+requests to the `/metrics` path itself; our `accessctr` counter tracks only requests to
+our application endpoints.
+
+== Or Download the Example Source
```

Comment:
STYLE: I suggest "(Optional) Download the Example Source Code"

Reply:
done